### DISCUSSION TITLE
Is there a way to write transactions with social login other than ERC20 ones? seems is not working for my custom contracts

### DISCUSSION THREAD

**Discussion by mbarr1111** [WalletConnect/walletconnect-monorepo#5535] [Developer Support]
Couldnt find documentation, but seems that only regular ERC20 transactions work when logged in with twitter or social, is that the case? how the user can interact with my contract when logged in with social account?

**Comment by mbarr1111** [comment]
very disappointed with reown, nobody responds discord, GitHub or whatever, they leave the users gothed

**Reply by arein** [reply]
Have you looked through our Discussions and Discord to validate your statement? You posted 4 days ago and it's Monday - we're not actively responding on the weekend

**Reply by mbarr1111** [reply]
discord is not helpful, the only interaction you get there is "good morning"

**Comment by arein** [comment]
@mbarr1111 there are plenty of apps out there using the social embedded wallet for all kinds of transactions - it's a wallet like any other. 

> how the user can interact with my contract when logged in with social account?

There will be a popup on the screen and the user can sign

Let us know if you have any issues

**Reply by mbarr1111** [reply]
hi thanks, but whenever I make a transaction to a custom smart contract we get:

{
    "status": "FAILED",
    "reasons": [
        {
            "field": "",
            "description": "Crypto utils error: Error when decoding ERC20 call: Unknown function selector."
        }
    ]
}


Works only when loggin with metamask or injected wallet, but social login throws always that, Im interacting with a method that is payable but we dont send value, when value is 0, it throws this.

**Reply by mbarr1111** [reply]
not sure why is this closed now?

**Reply by arein** [reply]
@mbarr1111  can you please provide proper steps to reproduce - is your website live?

**Reply by mbarr1111** [reply]
I will prepare the url , many thanks (the call to the orchestrator will be there)

**Reply by arein** [reply]
Amazing
Thank you @mbarr1111

**Reply by mbarr1111** [reply]
n ow works out for some reason, maybe a release? very weird but we are happy, no more erc20 Errors so far. (with the exact same code)

**Reply by arein** [reply]
great to hear it works no @mbarr1111 - can we see your website somewhere? We'd love to see what you've built
